frontend/spec/support/example_groups/partial_double_verification_example_group.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module PartialDoubleVerificationExampleGroup
```

Comment:
Can't we share the support file introduced in core via #3005 ? Maybe moving the module in `core/lib` into the `testing_support` folder?